src/components/NewItemForm.jsx

```diff
@@ -36,14 +55,41 @@ const NewItemForm = () => {
   };
 
   const handleDialogClose = (dialogState) => {
-    setAnchorEl(null);
+    handleMenuClose();
```

Comment:
For both vault collections and vault items, when a user closes the menu without submitting the form and opens up the model again, the form is still populated with the old values.

Reply:
Good catch! Updated